Celeste.Mod.mm/Patches/Decal.cs

```diff
@@ -116,6 +116,13 @@ private class DecalImage : Component {
             scaredAnimal = true;
         }
 
+        [MonoModIgnore]
+        private float frame;
```

Comment:
is this the right place for this, and am i using `[MonoModIgnore]` correctly?

Reply:
From a style standpoint, fields generally go at the top of the class. And I'm not sure if having [MonoModIgnore] hurts anything, but don't think it's necessary - if you look at the `scale` variable or the `patch_Player` class, you can generally add private fields into scope w/o issue